### DISCUSSION TITLE
InterpolateArraysRange - "Reverse" not working?

### DISCUSSION THREAD

**Discussion by franckaubry** [cables-gl/cables_docs#132] [! Welcome - readme first!]
If my understanding of the "InterpolateArraysRange" is correct, the "Reverse" button suppose to reverse the selection in the interpolation. If so, it seems broken in that it doesn't do anything.
![Cables_gl-InterpolateArraysRange -  Reverse not working](https://user-images.githubusercontent.com/97125854/151505867-0bdac893-0c20-4137-912b-ce62e769b24f.gif)

**Comment by tastez** [comment]
@franckaubry just so i understand fully before making an issue, you mean 
Reverse would reverse the interpolation that you selected in the drop down? Like Cubic Out would be Cubic in?

**Reply by franckaubry** [reply]
Ooooooh! Is the "Reverse" related to easing?
I thought it was related to where the selection in the array, meaning if it set to position "0", it would normally select from the beginning of the array, while selection "Reverse" would select from the end of the array. Meaning "Reverse" is a reversing of the array.
But I might have misunderstood what the actually idea of "Reverse" should be?
